examples/common/src/chariott/api.rs

```diff
@@ -290,7 +290,7 @@ impl<T: ChariottCommunication> Chariott for T {
                     .map(|v| {
                         Value::try_from(v).map_err(|_| Error::new("Could not parse read value."))
                     })
-                    // TODO: replace with common::OptionExt after #13
+                    // DevSkim: ignore DS176209 TODO: replace with common::OptionExt after #13
```

Comment:
Perhaps make these two separate comments?

Reply:
I actually removed this TODO. I don't think we plan on doing it anyway